common/tzif.ksy

```diff
@@ -0,0 +1,262 @@
+meta:
+  id: tzif
+  title: The Time Zone Information Format (TZif)
+  xref:
+    mime: application/tzif
+    # mime: "application/tzif-leap" if leap-second records are included
+    rfc: 8536
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: be
+doc: |
+  TZif is the binary file format used to represent data for a single time zone
+  compiled by the _zic_ compiler from information in the IANA time zone
+  database. It is used by most UNIX systems to calculate local time for a time
+  zone, usually identified by a time zone identifier string referring to a
+  relatively populous location within the geographic region covered by the time
+  zone, for example: "America/New_York" or "Europe/Paris". This identifier is
+  external to the TZif file itself.
+doc-ref: https://tools.ietf.org/html/rfc8536
+seq:
+  - id: v1_struct
+    type: tz_struct(false)
+  - id: v2_struct
+    type: tz_struct(true)
+    if: version >= 2
+  - id: v2_footer
+    type: tz_footer
+    if: version >= 2
+instances:
+  version:
+    value: v1_struct.header.version
+types:
+  tz_struct:
+    params:
+      - id: is_ts64
+        type: bool
+        doc: |
+          Chooses whether to use 64-bit or 32-bit time values. Set to `false`
+          for 32-bit time values (used in the version 1 data block). Set to
+          `true` for 64-bit time values (used in the version 2+ data block).
+    seq:
+      - id: header
+        type: tz_header
+      - id: data
+        type: tz_data(is_ts64)
+  tz_header:
+    doc-ref: https://tools.ietf.org/html/rfc8536#section-3.1
+    seq:
+      - id: magic
+        contents: TZif
+      - id: ver_byte
+        -orig-id: ver
+        type: u1
+        doc: |
+          File format version: 0x00 Version 1; 0x32 Version 2; 0x33 Version 3.
```

Comment:
should be either or a enum, or the undocumented internal structure decomposed anyhow.

Reply:
I don't want it to break if/when the version gets incremented in the future.

Reply:
There is an unreleased TZif version 4 that has no changes to the format, just some relaxations in the contents of the leap-second records (if they are present). I plan to update the doc-strings for v4 when it gets released officially.